xcode/Subconscious/Shared/Models/EntryStub.swift

```diff
@@ -7,6 +7,13 @@
 
 import Foundation
 
+struct AuthoredEntryStub: Identifiable, Equatable, Hashable {
```

Comment:
Annoying name, but this is accurate

Reply:
Done. It's an optional field now and `TranscludeView` can handle the case where it's missing (hides the pfp). Currently we always populate `author` eagerly but this leaves room to lazy-load author information in the future while rendering transcludes immediately.